src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +52,50 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
+      //ToDo(kristenmason@): display map
+      this.createButtons_();
+    })
+  }
+
+  /**
+   * Creates the Skip To Content and Return To Ad buttons once the user
+   *   grants permission to access their location and the map appears.
+   * @private 
+  */
+  createButtons_() {
+    const returnToAdButton = document.createElement("button");
+    returnToAdButton.innerHTML = "Return To Ad";
+    returnToAdButton.id = "returnToAd";
+    returnToAdButton.onclick = () => this.playAd_(returnToAdButton); 
+
+    const skipToContentButton = document.createElement("button");
+    skipToContentButton.innerHTML = "Skip To Content";
+    skipToContentButton.id = "skipToContent";
+    skipToContentButton.onclick = () => this.playContent_();
+
+    const container = document.getElementById('container');
```

Comment:
Do you think we could give this a more descriptive name than container? In another PR Kaleab suggested that in the html we call the div button_container for added clarity, I think it would be a good idea here as well

Reply:
the div I am referring to here is literally the entire iframe container so I can call it that if you'd prefer?